packages/address/src/address.ts

```diff
@@ -137,8 +139,7 @@ export default class Address extends AbstractAddress {
   }
 
   /**
-   * returns the address `checksum` as a string
-   *
+   * Returns the value of the `bech32Address` property
```

Comment:
`this.toChecksum()` doesn't return a `bech32Address`. Was this change a mistake?

Reply:
@nedsalk Good catch.

I think this change came by accident

Reply:
Technically it says that it returns the value of the property, not that it is a `bech32address`  , but I've updated the description to be more explicit 0568c2955